src/docs/forms/components/CalendarInput/examples/03-DisabledCalendarInput.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {CalendarInput} from "doj-react-adminlte/Form";
+
+export default class DisabledCalendarInput extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            disabledTrue: null,
+            disabledFalse: null
+        };
+    }
+
+    handleDateChange = (name, value) => {
+        this.setState({[name]: value});
+    };
+
+    render() {
+        const {disabledTrue,disabledFalse} = this.state;
+
+        return (
+            <div className="row">
+                <div className="col-xs-6">
+                    <CalendarInput name="disabledTrue" label="Disabled Calendar Input" value={disabledTrue}
```

Comment:
Same with the clearable calendar input example, add a checkbox to toggle the `disabled` flag.  Remove the other calendar input.